### PR TITLE
[REBASE] refactor: switch to new version of theme

### PR DESCRIPTION
**Pull request by Menci**

### CODE REVIEW

**Comment by Enter-tainer** [comment]
好像有个 pycache 被提交上来了

File: scripts/pre-build/install-theme.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ if [[ "$PREBUILD_NETLIFY" == "1" ]]; then
 fi
 
 if [ ! -d "$THEME_NAME" ] ; then
-  git clone --depth=1 "${THEME_REPO:-https://github.com/OI-wiki/mkdocs-material.git}" "$THEME_NAME"
+  git clone --depth=1 "${THEME_REPO:-https://github.com/Menci/mkdocs-material.git}" "$THEME_NAME"
```
Comment (Menci) [inline scripts/pre-build/install-theme.sh:15]: Revert it after merging https://github.com/OI-wiki/mkdocs-material/pull/38.

**Review by Enter-tainer — APPROVED** [review]
cool

**Comment by Enter-tainer** [comment]
@Ir1d  @billchenchina please review

**Review by Ir1d — APPROVED** [review]
辛苦啦！ 别忘了改 git clone 主题的路径 🙏

**Pull request merged**